rankers/BiMatchRanker/tests/test_bimatchranker.py

```diff
@@ -105,6 +43,38 @@ def create_data(query_chunk2match_chunk):
     return doc
 
 
+def test_bimatchranker():
+    query_chunk2match_chunk = {
+        100: [
+            {'parent_id': 1, 'id': 10, 'score': 0.4, 'length': 200},
+        ],
+        110: [
+            {'parent_id': 1, 'id': 10, 'score': 0.3, 'length': 200},
+            {'parent_id': 1, 'id': 11, 'score': 0.2, 'length': 200},
+            {'parent_id': 4294967294, 'id': 20, 'score': 0.1, 'length': 300},
+        ]
+    }
+
+    doc = create_data(query_chunk2match_chunk)
+    driver = SimpleChunk2DocRankDriver(
```

Comment:
is it possible to avoid using the driver here? we try to avoid depending from hub to the core.

Reply:
I feel like introducing driver with keep the same tests behaviour as before

Reply:
but it adds a dependency that is unwanted, can u mimic the behavior urself?

Reply:
So are we going to test then one scalar instead of the returned list? I'm also not sure how to do the other tests

Reply:
Ahhh sorry just saw last Joans comment. To make sure, we want to mimic getting the list without using a driver, right?

Reply:
And we lack integration tests for hub executors, test on a higher level will help us find out more potential bugs. Like the required_keys before. We both pass the tests in core and hub but still failed the examples.

Reply:
Ok I see, that's a good argument